src/Treeview.ts

```diff
@@ -70,7 +69,7 @@ export class JsonOutlineProvider implements vscode.TreeDataProvider<string> {
     public getChildren(element?: string): string[] {
         // check if there is a visible text editor
         if (vscode.window.visibleTextEditors.length > 0) {
-            if (isLanguageIdSupported(vscode.window.activeTextEditor.document.languageId)) {
+            if (vscode.window.activeTextEditor && this.isArmTemplateDocument(vscode.window.activeTextEditor.document)) {
```

Comment:
Is checking for the `activeTextEditor` necessary here?  I'm just asking since this is a newly introduced behavior.

Reply:
I dereference it in the next part of the expression, so yes.